todo_app/templates/index.html

```diff
@@ -11,8 +11,20 @@ <h1 class="display-4">To-Do App</h1>
     <div class="col-md-12">
       <h2>Items</h2>
       <ul class="list-group mb-4">
-        <!-- List the todo items here -->
+        {% for item in items %}
+        <ui>{{item.title}}</ui>
+        {% endfor %}
       </ul>
     </div>
   </div>
+  <div class="row">
+    <div class="col-md-12">
+      <h2>Add Item</h2>
+      <form action="{{url_for('add')}}" method="POST">
+        <label for="title">Title:</label>
+        <input type="text" id="title" name="title">
```

Comment:
I'd add a `required` attribute here so that you cannot add empty todos:
```suggestion
        <input type="text" id="title" name="title" required>
```